http/src/main/resources/init-resources/startup.sh

```diff
@@ -226,9 +232,7 @@ if [[ "${CLOUD_SERVICE}" == 'GCE' ]]; then
     # (1/6/22) Restart Jupyter Container to reset `NOTEBOOKS_DIR` for existing runtimes. This code can probably be removed after a year
     if [ ! -z "$JUPYTER_DOCKER_IMAGE" ] ; then
         echo "Restarting Jupyter Container $GOOGLE_PROJECT / $CLUSTER_NAME..."
-
-        # Make sure when runtimes restarts, they'll get a new version of jupyter docker compose file
-        $GSUTIL_CMD cp gs://${INIT_BUCKET_NAME}/`basename ${JUPYTER_DOCKER_COMPOSE}` $JUPYTER_DOCKER_COMPOSE
```

Comment:
Turns out that we do not need to always pull the latest docker compose file, we only need to inject updated environment variables.